PatientWebApplication/SearchMicroserviceApi/Dtos/MicroserviceReferralDto.cs

```diff
@@ -0,0 +1,41 @@
+﻿using SearchMicroserviceApi.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SearchMicroserviceApi.Dtos
+{
+    public class MicroserviceReferralDto : Entity
+    {
+        public string medicine { get; set; }
+        public string takeMedicineUntil { get; set; }
+        public int quantityPerDay { get; set; }
+        public string classify { get; set; }
+        public string comment { get; set; }
```

Comment:
Write class properties to start with upper letter and remove "this" in constructors